js/react/utils/constants.jsx

```diff
@@ -4,4 +4,57 @@ const SIGNUP = 'SIGNUP';
 
 const GOOGLE_CLIENT_LIBRARY_URL = 'https://accounts.google.com/gsi/client';
```

Comment:
this can be a simple `.js` file, no need for a `.jsx` extension

Reply:
it was an existing file, but ok I will change it

Reply:
sorry, must have missed that in a previous review